frb_codegen/src/library/commands/cargo_expand.rs

```diff
@@ -84,12 +93,14 @@ fn run_cargo_expand(
     // let _pb = simple_progress("Run cargo-expand".to_owned(), 1);
     debug!("Running cargo expand in '{rust_crate_dir:?}'");
 
-    let args = vec![
-        PathBuf::from("expand"),
-        PathBuf::from("--lib"),
-        PathBuf::from("--theme=none"),
-        PathBuf::from("--ugly"),
-    ];
+    let args = command_args!(
+        "expand",
+        "--lib",
+        "--theme=none",
+        "--ugly",
+        "--config",
+        r#"build.rustflags="--cfg frb_expand""#
```

Comment:
nit: as mentioned earlier, it would be great to do something like:
```
impl CachedCargoExpand {
  fn execute_awaring_frb() {
    unwrap_frb_attrs_in_doc(self.execute(extra_args = r#"build.rustflags="--cfg frb_expand""#))
  }
}
```

and let the inner code unaware of frb